packages/frontend-2/components/viewer/saved-views/panel/views/Group.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+  <LayoutDisclosure v-model:open="open" :title="group.title" lazy-load>
+    <div>
+      <div v-if="isVeryFirstLoading" class="flex justify-center">
+        <CommonLoadingIcon class="m-4" />
+      </div>
+      <div v-else>
+        <div
+          v-if="views.length"
+          class="flex flex-col gap-3 max-h-64 overflow-y-auto simple-scrollbar"
+        >
+          <ViewerSavedViewsPanelView
+            v-for="view in views"
+            :key="view.id"
+            :view="view"
+          ></ViewerSavedViewsPanelView>
+        </div>
+        <InfiniteLoading
+          v-if="views.length"
+          :settings="{ identifier }"
+          hide-when-complete
+          @infinite="onInfiniteLoad"
+        />
+      </div>
+    </div>
+  </LayoutDisclosure>
+</template>
+<script setup lang="ts">
+import { omit } from 'lodash-es'
+import { usePaginatedQuery } from '~/lib/common/composables/graphql'
+import { graphql } from '~/lib/common/generated/gql'
+import type { ViewerSavedViewsPanelViewsGroup_SavedViewGroupFragment } from '~/lib/common/generated/gql/graphql'
+import { useInjectedViewerState } from '~/lib/viewer/composables/setup'
+
+graphql(`
+  fragment ViewerSavedViewsPanelViewsGroup_SavedViewGroup on SavedViewGroup {
```

Comment:
one fragment we get from ancestor query, the other we use in the paginated view query that runs from this query